doc/attributes.xxml

```diff
@@ -17,12 +17,21 @@
 <!-- doxrox-include igraph_attribute_type_t -->
 </section>
 
+<section id="attribute-combinations">
+<title>Handling attribute combination lists</title>
+<!-- doxrox-include about_attribute_combination -->
+<!-- doxrox-include igraph_attribute_combination_init -->
+<!-- doxrox-include igraph_attribute_combination_add -->
+<!-- doxrox-include igraph_attribute_combination_remove -->
+<!-- doxrox-include igraph_attribute_combination_destroy -->
+<!-- doxrox-include igraph_attribute_combination_type_t -->
+</section>
+
 <section id="accessing-attributes-from-c">
 <title>Accessing attributes from C</title>
-
 <!-- doxrox-include cattributes -->
 
-<section id="query-attributes"><title>Query attributes</title>
+<section id="query-attributes"><title>Query C attributes</title>
```

Comment:
I would leave this as is; all the titles are in a section titled "Accessing attributes from C" so there is no need to emphasize again that we are dealing with C attributes.